Donation-platform/src/main/java/com/google/FindTrendingCharities.java

```diff
@@ -67,31 +69,36 @@
     // and avgReview is a weighted average of the userRating and the charityNavigatory API rating
     // weights for the averages are stored as class constants 
     private double calcCharityTrendingScore(Charity charity) {
-        boolean hasCharityNavRating = false;
         double charityNavRating, scaledCharityNavRating, userRating, avgReview;
```

Comment:
As per this https://stackoverflow.com/questions/1411463/in-java-should-variables-be-declared-at-the-top-of-a-function-or-as-theyre-ne declare variable close to where they are being used, in this case you can directly declare where you are initializing except for the case where it goes out of scope.

Reply:
fixed